### DISCUSSION TITLE
Timeline component

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4086] [Ideas]
we are just missing a Timeline component, like https://vuetifyjs.com/en/components/timelines/

are you planning to incorporate something alike? or i'll try to add it! (hoping the CSS won't conflict)

**Comment by Mister-Hope** [comment]
It would be great if you can open a pr  And add it in plugin-components pkg

Furthermore, you can also try to add a step component,  It should  accept a set of slots where each slot is a step item,  Prev and next button is below each step content where users can go through a set of actions within a small space in the page

**Reply by StefanoCecere** [reply]
ok, first i have to deal with the build error.. opening a new issue
